Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-alpine
+FROM php:7.4-buster
```

Comment:
shouldn't this file go in `7.4` subfolder ?

Reply:
I did not move it - I assume that this means something like `latest` - check the GH Actions for ex. in the `atk4/ui` repo - there are workflows for `7.2`, `7.3` and `latest`.

So I expect, when this line will be changed in future to `8.0`, the workflow for `latest` will use it (thus it makes sense to keep it like it is (or duplicate it to offer `7.4` beside `latest` as well)).

Reply:
OK. Probably you're right. Let @romaninsh check this.